ULWGL/ulwgl_consts.py

```diff
@@ -26,16 +26,3 @@ class Color(Enum):
     "getcompatpath",
     "getnativepath",
 }
-
-USAGE = """
-example usage:
-  GAMEID= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= ulwgl-run /home/foo/example.exe -opengl
-  WINEPREFIX= GAMEID= PROTONPATH= ulwgl-run ""
-  WINEPREFIX= GAMEID= PROTONPATH= PROTON_VERB= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= STORE= ulwgl-run /home/foo/example.exe
-  ULWGL_LOG= GAMEID= ulwgl-run /home/foo/example.exe
-  ulwgl-run --config /home/foo/example.toml
-"""
```

Comment:
My opinion is that this should not be removed. These are quick examples, manpage can go in depth about various things about ULWGL as a whole. such as information about the runtime. IMHO each serve their own purpose.

Reply:
I actually don't share that opinion, and my original intention was actually a man page from the beginning. The usage was just a stand-in until there was man page created, which is now.

However, if this is really a desired feature for the minority of CLI users, I'll be willing to bring back the usage.